### PR TITLE
HYACINTH-842 batchImports GQL bug on separate failures

### PR DESCRIPTION
**Pull request by barmintor**

### CODE REVIEW

File: app/javascript/src/hyacinth_v1/components/batch_imports/BatchImportShow.js [hunk 1]
```diff
@@ -47,7 +47,10 @@ function BatchImportShow() {
     batchImport: {
       numberOfInProgressImports: inProgress,
       numberOfPendingImports: pending,
-      numberOfFailureImports: failure,
+      numberOfCreationFailureImports: creationFailure,
+      numberOfUpdateFailureImports: updateFailure,
+      numberOfPersistFailureImports: persistFailure,
+      numberOfPublishFailureImports: publishFailure,
```
Comment (elohanlon) [inline app/javascript/src/hyacinth_v1/components/batch_imports/BatchImportShow.js:53]: Do you mind making these variable names plural?  e.g. creationFailure -> creationFailures

Comment (barmintor) [reply app/javascript/src/hyacinth_v1/components/batch_imports/BatchImportShow.js:53]: I don't! I was just trying to follow the pattern of the predecessor variable.

File: app/javascript/src/hyacinth_v1/components/batch_imports/digital_object_imports/DigitalObjectImportIndex.js [hunk 2]
```diff
@@ -69,13 +69,18 @@ function DigitalObjectImportIndex() {
       status,
       originalFilename,
       numberOfPendingImports: pending,
-      numberOfFailureImports: failure,
+      numberOfCreationFailureImports: creationFailure,
+      numberOfUpdateFailureImports: updateFailure,
+      numberOfPersistFailureImports: persistFailure,
+      numberOfPublishFailureImports: publishFailure,
```
Comment (elohanlon) [inline app/javascript/src/hyacinth_v1/components/batch_imports/digital_object_imports/DigitalObjectImportIndex.js:75]: Same note here about plurals.

File: app/javascript/src/hyacinth_v1/components/batch_imports/BatchImportShow.js (hunk 1 above)
Comment (elohanlon) [reply app/javascript/src/hyacinth_v1/components/batch_imports/BatchImportShow.js:53]: Ben and I talked this over and we'll go with `Count` instead:

inProgressCount, pendingCount, creationFailureCount, etc.

**Review by elohanlon — APPROVED** [review]

**Pull request merged**